### DISCUSSION TITLE
CURLOPT_PROXY authentication

### DISCUSSION THREAD

**Discussion by t1m013y** [curl/curl#15802] [Q&A] [answered]
Can `CURLOPT_PROXY` be used for authentication with username and password given in proxy string, for example: `curl_easy_setopt(curl, CURLOPT_PROXY, "socks5://username:password@XXX.XXX.XXX.XXX:80");`, where `XXX.XXX.XXX.XXX` is proxy IP?

**Comment by bagder** [comment, marked as answer]
Yes, that is supported. I'll make sure it gets mentioned in the manpage.
